soc/nxp/mcx/mcxa/Kconfig.soc

```diff
@@ -90,3 +90,11 @@ config SOC_PART_NUMBER
 	default "MCXA276VLL" if SOC_PART_NUMBER_MCXA276VLL
 	default "MCXA276VLH" if SOC_PART_NUMBER_MCXA276VLH
 	default "MCXA276VPN" if SOC_PART_NUMBER_MCXA276VPN
+
+config SOC_WAKEUP_DELAY
+	int "Wakeup delay"
+	default 91
+	help
+	  When a different VDD_CORE level is configured in low
+	  power modes compared to active modes, the wakeup delay
+	  must be configured to a minimum value.
```

Comment:
this config should not be added in a Kconfig.soc file, Kconfig.soc is only for HWM hierarchy description, put in Kconfig file and namespace for the platform